metricbeat/scripts/module/metricset/fields.yml

```
@@ -1,5 +1,6 @@
 - name: {metricset}
   type: group
+  release: experimental
```

I'd suggest to avoid use of experimental by default.
```suggestion
  release: beta
```